modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1193 @@
+
+The new comment sidebar text-entry field contains placeholder text.
+
+Previously, this placeholder string was, in English, `Say something…`.
+
+For this update, the English-language string has been changed to `Add comment…`.
+
+Updated translations of this placeholder text have also been added to all (productname} supported languages.
+
+
+==== The label name for the Save Comment button has been changed and can now be translated
+
+Previously, the Save Comment was labeled *Save*. After field reports that end-users were interpreting this to mean the entire {productname} document would be saved on pressing this button, two changes were made.
+
+1. The default label for this button is now *Comment*.
++
+This new default presents for both new comments and replies to existing comments.
+
+2. This default label can now be translated.
+
+As is usual in {productname}, translation of this string is done wth `addI18n()`:
+
+[source,js]
+----
+tinymce.addI18n('<optional-IETF-language-code>', {
+   'Comment': '<translation>'
+});
+----
+
+NOTE: When a comment is being edited, the default button, which writes the changes back and closes the Edit Comment dialog, is still labeled *Save*.
+
+
+==== The `change` event was not fired when deleting all conversations
+
+Previously, when all comments were deleted, existing undo levels were re-written but no new undo level was created. As a consequence no `change` event was fired.
+
+This resulted in applications that relied on {productname} reporting its current state incorrectly assuming the now-deleted comments still existed.
+
+This update corrects this. In Comments 3.1.0, deleting any or all comments fires a `change` event. And a `changed` message is, consequently, printed to the console.
+
+
+==== An additional undo level was incorrectly created when the Comments plugin was loaded at {productname} initialization
+
+Previously, when the Comments plugin loaded, a bug in the plugin caused {productname} to put the comment model data into a first undo level.
+
+This caused {productname} to present as if an undo option was available when there was no action to undo (for example, the Undo button would enable if all that had changed as the insertion point moving).
+
+This update fixes this bug. When the Comments plugin loads, the comment model data is no longer placed into an undo level and no Undo actions will present as available until there are actual changes available to undo.
+
+
+==== Opening the comments sidebar via an editor command did not respect the `skip_focus` argument
+
+If the Comments sidebar was opened with an editor command and `skip_focus` was set to true — `editor.execCommand("ToggleSidebar", false, "showcomments", { skip_focus: true });` — the sidebar still stole focus.
+
+This update corrects this: `skip_focus: true` is honored, as expected, and the sidebar does not steal focus.
+
+
+==== Comments did not blocking the Save UI while saving the data in callback mode
```

Comment:
```suggestion
==== Comments did not blocking the sidebar UI while saving the data in callback mode
```
I think this is a little clearer? as technically it could happen both when replying and saving edits.

Reply:
Corrected this locally and pushed because the gerund needed editing as well: s/blocking/block/.